neqo-common/src/udp.rs

```diff
@@ -0,0 +1,155 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![allow(clippy::missing_errors_doc)]
+#![allow(clippy::missing_panics_doc)]
```

Comment:
I don't think these docs would be helpful, since each of the functions just delegate to `tokio` and `quinn-udp`. I doubt it would ever be complete, nor up-to-date for long. Happy to add it in case you consider it helpful.

Reply:
Can you put that text (or a shorter version thereof) in a comment?